text/0083-rawkv-cross-cluster-replication.md

```diff
@@ -0,0 +1,164 @@
+# RawKV Cross Cluster Replication
+
+- RFC PR: https://github.com/tikv/rfcs/pull/86
+- Tracking Issue: https://github.com/tikv/repo/issues/0000
+
+## Summary
+
+This proposal introduces the technical design of RawKV Cross Cluster Replication.
+
+## Motivation
+
+Customers are deploying TiKV clusters as non-transaction Key-Value (RawKV) storage. But the lack of *Cross Cluster Replication* is a obstacle to deploy disaster recovery clusters for providing more highly available services.
+
+The ability of cross cluster replication will help TiKV be more adoptable to industries such as Banking and Finance.
```

Comment:
Could you list key challenges of the use case "Banking" plus "Cross Cluster Replication"? It would be helpful for understanding the design in the RFC.

Here is my initial thought,
* How to address high latency issue between clusters?
* How do we define the consistency of raw kv?

Reply:
@overvenus I add a chapter about "Key Challenges & Solutions", and specify that we will implement RawKV features in TiCDC as new modules for isolation with original TiDB related logics.
PTAL, thanks~